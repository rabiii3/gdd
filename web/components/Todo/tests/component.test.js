import React from 'react';
import { shallow } from 'enzyme';
import {Todo} from '..';

describe('web | components | Todo | component', () => {

  describe('render', () => {
  it('should match snapshot', () => {
  const props = {
  comment: {},
  people: {},
  classes: {},
  values: {},
  fullWidth: false,
  width: "",
  adminsOrHeadHunters: [],
  };
  const wrapper = shallow(<Todo {...props} />);
  expect(wrapper).toMatchSnapshot();
  });

   
  });
});
