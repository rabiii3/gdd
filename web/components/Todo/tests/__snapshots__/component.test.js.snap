// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`web | components | Todo | component render should match snapshot 1`] = `
<WithStyles(Grid)
  alignitem="center"
  container={true}
  direction="row"
  justify="space-between"
>
  <WithStyles(Grid)
    item={true}
    lg={4}
    xs={12}
  >
    <ReactFinalForm(Field)
      className=""
      component={[Function]}
      label={
        <label>
           
          <FormattedMessage
            defaultMessage="What"
            id="components.Todo.what"
            values={Object {}}
          />
           
        </label>
      }
      name="what"
    >
      <WithStyles(MenuItem)
        key="none"
        value="none"
      >
        <WithStyles(Grid)
          alignitem="center"
          container={true}
          direction="row"
          justify="flex-start"
        >
          <pure(BlockIcon) />
          <Spacer
            size="SMALL"
          />
          None
        </WithStyles(Grid)>
      </WithStyles(MenuItem)>
      <WithStyles(MenuItem)
        key="call"
        value="call"
      >
        <WithStyles(Grid)
          alignitem="center"
          container={true}
          direction="row"
          justify="flex-start"
        >
          <pure(CallIcon) />
          <Spacer
            size="SMALL"
          />
          Call
        </WithStyles(Grid)>
      </WithStyles(MenuItem)>
      <WithStyles(MenuItem)
        key="forward"
        value="forward"
      >
        <WithStyles(Grid)
          alignitem="center"
          container={true}
          direction="row"
          justify="flex-start"
        >
          <pure(ForwardIcon) />
          <Spacer
            size="SMALL"
          />
          Forward
        </WithStyles(Grid)>
      </WithStyles(MenuItem)>
      <WithStyles(MenuItem)
        key="meet"
        value="meet"
      >
        <WithStyles(Grid)
          alignitem="center"
          container={true}
          direction="row"
          justify="flex-start"
        >
          <pure(FaceIcon) />
          <Spacer
            size="SMALL"
          />
          Meet
        </WithStyles(Grid)>
      </WithStyles(MenuItem)>
    </ReactFinalForm(Field)>
  </WithStyles(Grid)>
  <WithStyles(Grid)
    item={true}
    lg={4}
    xs={12}
  >
    <WithStyles(Grid)
      alignItems="flex-start"
      container={true}
      direction="column"
      justify="flex-end"
    >
      <ReactFinalForm(Field)
        className=""
        component={[Function]}
        keyboard={true}
        name="when"
      />
    </WithStyles(Grid)>
  </WithStyles(Grid)>
  <WithStyles(Grid)
    item={true}
    lg={4}
    xs={12}
  >
    <WithStyles(Grid)
      alignItems="flex-start"
      container={true}
      direction="column"
      justify="flex-end"
    >
      <WithStyles(Grid)
        item={true}
      />
      <WithStyles(Grid)
        item={true}
      >
        <ReactFinalForm(Field)
          className=""
          component={[Function]}
          defaultValue={Array []}
          isMulti={true}
          name="who"
          placeholder={
            <FormattedMessage
              defaultMessage="Who"
              id="components.Todo.who"
              values={Object {}}
            />
          }
          suggestion={Array []}
          textFieldProps={
            Object {
              "InputLabelProps": Object {
                "shrink": true,
              },
            }
          }
          type="select"
        />
      </WithStyles(Grid)>
    </WithStyles(Grid)>
  </WithStyles(Grid)>
</WithStyles(Grid)>
`;
